app/mailers/devise_mailer.rb

```diff
@@ -22,6 +22,9 @@ def headers_for(action, opts)
     headers[:body] = render_to_string(
       inline: @template.body
     ) unless @template.body.nil?
+    headers[:reply_to] = render_to_string(
+      inline: @template.reply_to
```

Comment:
Ao invés de termos reply_to no EmailTemplate, acho melhor sempre acessar de CustomVariable. A não ser que tenha a opção de editar o reply_to na interface do template e isso seja algo opcional.

Da forma que está, se EmailTemplate for salvo com um reply_to específico (carregado automaticamente de CustomVariable, já que não há opção de editar) e o CustomVariable mudar, o valor que está em EmailTemplate provavelmente não vai mudar e vai causar um bug.